toolkit/sharedlib/src/main/java/com/arcgismaps/toolkit/ui/expandablecard/theme/ExpandableCardTheme.kt

```diff
@@ -0,0 +1,166 @@
+     */
+    @Suppress("unused")
+    val typography: ExpandableCardTypography
+        @Composable
+        @ReadOnlyComposable
+        get() = LocalTypography.current
+
+    /**
+     * Retrieves the current [ExpandableCardShapes].
+     */
+    val shapes: ExpandableCardShapes
+        @Composable
+        @ReadOnlyComposable
+        get() = LocalShapes.current
+}
+
+/**
+ * Provides a default [ExpandableCardTheme] to the given [content] so that the ExpandableCard can be
+ * customized.
+ *
+ * The default value for the [colorScheme] and [typography] is based on the current [MaterialTheme].
+ * See [ExpandableCardDefaults.colorScheme], [ExpandableCardDefaults.typography], and
+ * [ExpandableCardDefaults.shapes] for the exact configuration used.
+ *
+ * @param colorScheme An [ExpandableCardColorScheme] to use for this compose hierarchy.
+ * Default is [ExpandableCardDefaults.colorScheme].
+ * @param typography An [ExpandableCardTypography] to use for this compose hierarchy.
+ * Default is [ExpandableCardDefaults.typography].
+ * @param shapes An [ExpandableCardShapes] to use for this compose hierarchy.
+ * Default is [ExpandableCardDefaults.shapes].
+ * @param content The content to which the theme should be applied.
+ */
+@Composable
+fun ExpandableCardTheme(
+    colorScheme: ExpandableCardColorScheme = ExpandableCardDefaults.colorScheme(),
+    typography: ExpandableCardTypography = ExpandableCardDefaults.typography(),
+    shapes: ExpandableCardShapes = ExpandableCardDefaults.shapes(),
+    content: @Composable () -> Unit
+) {
+    CompositionLocalProvider(
+        LocalColorScheme provides colorScheme,
+        LocalTypography provides typography,
+        LocalShapes provides shapes
+    ) {
+        content()
+    }
+}
+
+/**
+ * A color scheme that holds all the color parameters for a [ExpandableCard].
+ *
+ * The scheme provides default values for all colors as a starting point for customization. These
+ * defaults are populated using [MaterialTheme].
+ *
+ * Use [ExpandableCardDefaults.colorScheme] to create a new instance with the default values.
+ *
+ * @property headerTextColor The color scheme for the ExpandableCard Header
+ * @property containerColor The color scheme for the ExpandableCard Body
+ * @since 200.6.0
+ */
+@Immutable
+data class ExpandableCardColorScheme internal constructor(
+    val headerTextColor: Color,
+    val containerColor: Color,
+    val galleryContainerColor: Color,
+    val borderColor: Color
+)
+/**
+ * A Typography system for the [ExpandableCard] built on top of [MaterialTheme].
+ *
+ * Use [ExpandableCardDefaults.typography] to create a new instance with the default values.
+ *
+ * @property headerStyle The typography for the ExpandableCard Header.
+ * @property bodyStyle The typography for the ExpandableCard Body.
+ * @since 200.6.0
+ */
+@Immutable
+class ExpandableCardTypography internal constructor(
+    val headerStyle: TextStyle,
+    val bodyStyle: TextStyle
```

Comment:
```suggestion
    val headerStyle: TextStyle,
    val bodyStyle: TextStyle
```
Similarly, these are not utilized in `ExpandableCard`.